libbeat/common/logging.go

```diff
@@ -27,13 +29,14 @@ var maskList = MakeStringSet(
 	"urls",
 	"host",
 	"hosts",
+	"authorization",
```

Comment:
Let's add `proxy-authorization` as well. We have a few inputs (including heartbeat) that allow users to configure proxies.

Reply:
Done 👍🏽